rustls/src/msgs/handshake.rs

```diff
@@ -1308,7 +1348,8 @@ pub struct ServerHelloPayload {
     pub(crate) session_id: SessionId,
     pub(crate) cipher_suite: CipherSuite,
     pub(crate) compression_method: Compression,
-    pub(crate) extensions: Vec<ServerExtension>,
+
+    pub extensions: Vec<ServerExtension>,
```

Comment:
I think our ordering style guide would want this field promoted to the first entry in the `struct` now that it's `pub`.